### PR TITLE
Support fetching libraries from local directory

### PR DESCRIPTION
**Pull request by psss**
This allows easier beakerlib library development.

### CODE REVIEW

File: spec/tests/require.fmf [hunk 1]
```diff
@@ -28,6 +28,10 @@ description: |
     ``destination`` to choose a different location. The ``nick``
     key can be used to override the default git repository name.
 
+    For debugging beakerlib libraries it is useful to reference
+    the development version directly from the local filesystem.
+    Use the ``path`` key to specify a full path to the library.
```
Comment (lukaszachy) [inline spec/tests/require.fmf:33]: Conflicting name with "path" from https://fmf.readthedocs.io/en/stable/concept.html#identifiers. Currently tmt doesn't support it( #446)

How about 'location' ?

Comment (lukaszachy) [reply spec/tests/require.fmf:33]: Ah, you are right - absolute vs relative.

Comment (psss) [reply spec/tests/require.fmf:33]: Good to merge then?

**Review by lukaszachy — APPROVED** [review]

**Comment by psss** [comment]
@thrix, fyi, another `Couldn't resolve host name` error on [fedora-34](http://artifacts.dev.testing-farm.io/e38bd2e5-e833-4042-821d-281f81e12bb1/).

**Comment by psss** [comment]
@sopos, fyi, thanks to this, developing of beakerlib libraries should be much easier.

**Pull request merged**